source/authentication.txt

```diff
@@ -0,0 +1,285 @@
+   human users as "Workforce Identity" and applications as 
+   "Workload Identity".
+
+The following table provides guidance on the recommended authentication methods 
+for staging and production ("upper") environments, and other methods you might 
+use in development and test ("lower") environments. 
+
+.. list-table:: 
+   :widths: 10 13 40 37
+   :header-rows: 1
+
+   * - Resource
+     - User 
+     - Recommendations for Production
+     - Options for Dev and Test
+
+   * - Atlas UI 
+     - Workforce (Human)
+     - | Use :ref:`Federated Auth <arch-center-federated_auth>` with an SAML 2.0 
+         identity provider, such as Okta, Microsoft Entra ID, or Ping Identity. 
+       | -- *or* --
+       | Use :ref:`Multi-factor Authentication (MFA) <arch-center-mfa>`.
+       | -- *and* --
+       | Use IP access restrictions.
+     - | Use Atlas credentials and MFA.
+    
+   * - Atlas Database
+     - Workforce (Human)
+     - Use :ref:`Workforce Identity Federation <arch-center-workforce_if>`
+     - | Use :ref:`X.509 <arch-center-x509>` certificates or :ref:`SCRAM <arch-center-scram>`.
+       | Consider :ref:`creating temporary database users <arch-center-just-in-time>`
+         with just-in-time database access.
+    
+   * - Atlas Database
+     - Workload (Application)
+     - | Use :ref:`Workload Identity Federation <arch-center-workload_if>`.
+       | -- *or* --
+       | Use :ref:`AWS-IAM authentication <arch-center-iam>`.
+       | -- *or* --
+       | Use :ref:`X.509 <arch-center-x509>` certificates
+     - :ref:`X.509 <arch-center-x509>` certificates or :ref:`SCRAM <arch-center-scram>`
+
+   * - Atlas APIs
+     - Workforce (Human)
+     - Use :ref:`<arch-center-admin-service-accounts>`.
+     - Use :ref:`<arch-center-admin-service-accounts>` or 
+       :ref:`<arch-center-admin-api-keys>`.
+
+   * - Atlas APIs
+     - Workload (Application)
+     - Use :ref:`<arch-center-admin-service-accounts>`.
+     - :ref:`<arch-center-admin-api-keys>`
+
+.. collapsible::
+   :heading: Single-Region Deployment Recommendations
+   :sub_heading: Recommendations that apply only to deployments in a single region
+   :expanded: true
+
+   <<Recommendations that apply only to deployments in a single region here>>
+
+.. collapsible::
+   :heading: Multi-Region and Multi-Cloud Deployment Recommendations
+   :sub_heading: Recommendations that apply only to deployments across multiple regions or multiple cloud providers
+   :expanded: true
+
+   <<Recommendations that apply only to deployments across multiple regions or multiple cloud providers here>>
+
+
+The following sections provide details about the authentication methods. 
+
+.. _arch-center-federated_auth:
+
+Federated Authentication
+~~~~~~~~~~~~~~~~~~~~~~~~
+Federated authentication allows you to manage all authentication to
+the {+atlas-ui+} across multiple systems and applications through a
+central identity provider. You can use any :abbr:`SAML (Security Assertion
```

Comment:
does this need to be SAML 2.0? I'm not super close to the specifics here on whether it matters